index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css"/>
+</head>
+<body>
+    <nav class='navigation-bar' >
+        <div class='list-parent-left' id='symbol'>
+            <a class='li-link'href='index.html'> <h3>Chikku<span style='color:rgb(10, 177, 10)'>Ui</span></h3> </a>  
+        </div>
+        <div class='list-parent-right' id='modal'>
+            <a  href="javascript:void(0);" class="li-icon" onclick="modelFunction()">
+                <i class="fa fa-bars"></i>
+              </a>
+              <a class='li-link' href='index.html'> Home</a>
+              <a class='li-link' href='components/alert/index.html'>Documentation</a>
+              <a class='li-link' href='https://github.com/vineeth-t/Main/tree/Devlopment'>Github</a>
+              <!-- <a class='li-link'href='#'>V0.1</a> -->
+        </div>
+        </nav>
+<div class='center-div'>
+   <h2 style='text-align: center;margin: 1rem;'>Plug & Play <span style='color:rgb(10, 177, 10)'>CSS</span></h2>
+   <p style='font-size: large;font-size: 1rem;'>Css is very powerful tool in Web Development.This is my version of CSS which is easy to Plug and Play for any kind of projects.</p><br/>
+   <div style='text-align: center'> 
```

Comment:
I think it is not a good practice to use inline styling which you have used here, try changing this if possible.